_includes/head.html

```diff
@@ -6,14 +6,14 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Open Source Design</title>
+    <title>Open Source Design events</title>
```

Comment:
* why not capitalize "events"? 
* Also, do we plan to have this 'standard' everywhere? Since if we change this title, we will have inconsistencies with /articles, /jobs, etc.